### DISCUSSION TITLE
Capturing Logout Activity

### DISCUSSION THREAD

**Discussion by skmony** [directus/directus#22283] [Feature Requests]
### Summary

Need a feature to capture, logout event like we are capturing Login event, Create, Edit, Delete, Comment on the Activity feed

### Motivation

We should keep a track of when a user logs out of the application 

### Detailed Design

Need a feature to capture, logout event like we are capturing Login event, Create, Edit, Delete, Comment on the Activity feed

### Must Have:
- A Capture Logout event when a user logs out from Directus application
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

NA

### Alternatives

Logout activity is not getting logged as of now.

### Adoption Strategy

It will improve the activity monitoring of the application.

**Comment by br41nslug** [comment]
Your detailed design could use some more detail 😬 How would these events deal with expired tokens? which i suspect is a very common situation where people get "logged out"

**Reply by skmony** [reply]
For my use case I only need to capture the event when a user logs out manually.
